app/src/main/java/eu/darken/capod/monitor/ui/MonitorNotifications.kt

```diff
@@ -71,9 +76,37 @@ class MonitorNotifications @Inject constructor(
         }
 
         return builder.apply {
+            @StringRes var contentTitleRes = eu.darken.capod.common.R.string.pods_case_unknown_state
+            // Options here should be mutually exclusive, and are prioritized by their order of importance
+            // Some options are omitted here, as they will conflict with other options
+            // TODO: Implement a settings pane to allow user to customize this
+            device.apply {
```

Comment:
Nitpicking: Maybe a `when` here instead of apply+early return, but it's alright :heart: